tasks/marketplacesvm-push-disk-images/tests/test-marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,190 @@
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+
+              cat > "$(workspaces.data.path)/snapshot_spec.json" << EOF
+              {
+                "application": "amd-bootc-1-3-raw-disk-image",
+                "artifacts": {},
+                "components": [
+                  {
+                    "containerImage": "quay.io/workload/tenant/test-product/amd-bootc-1-3-raw-disk-image@sha256:123456",
+                    "name": "amd-bootc-1-3-raw-disk-image",
+                    "source": {
+                      "git": {
+                        "revision": "1abbfcdbc1c5e8b7aba07673297237ed192b50e2",
+                        "url": "https://gitlab.com/konflux/test-product/disk-images/amd-bootc"
+                      }
+                    }
+                    "staged": {
+                      "destination": "test-product-1_DOT_3-x86_64-isos",
+                      "files": [
+                        {
+                          "filename": "test-product-amd-1.3-1732045201-x86_64.raw",
+                          "source": "disk.raw"
+                        }
+                      ],
+                      "version": "1.3"
+                    },
+                    "productInfo": {
+                      "filePrefix": "test-product-amd-1.3",
+                      "productCode": "TESTPRDOCUT",
+                      "productName": "Test Product",
+                      "productVersionName": "1.3"
+                    },
+                    "starmap": [
+                      {
+                        "cloud": "aws",
+                        "mappings": {
+                          "aws-emea": {
+                            "destinations": [
+                              {
+                                "architecture": "x86_64",
+                                "destination": "00000000-0000-0000-0000-000000000000",
+                                "overwrite": false,
+                                "restrict_version": true
+                              }
+                            ],
+                            "provider": null
+                          },
+                          "aws-na": {
+                            "destinations": [
+                              {
+                                "architecture": "x86_64",
+                                "destination": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+                                "overwrite": false,
+                                "restrict_version": true
+                              }
+                            ],
+                            "provider": null
+                          }
+                        },
+                        "name": "test-product",
+                        "workflow": "stratosphere"
+                      }
+                    ]
+                  }
+                ]
+              }
+              EOF
+
+              cat > "$(workspaces.data.path)/starmap.json" << EOF
```

Comment:
This file isn't passed to the task. Why is it here?

Reply:
Just to compare the expected results in the tests